hexa/workspaces/schema/types.py

```diff
@@ -96,6 +102,26 @@ def resolve_workspace_members(workspace: Workspace, info, **kwargs):
     )
 
 
+@workspace_object.field("invitations")
+def resolve_workspace_invitations(workspace: Workspace, info, **kwargs):
+    request: HttpRequest = info.context["request"]
+
+    status = [WorkspaceInvitationStatus.ACCEPTED, WorkspaceInvitationStatus.PENDING]
+    if kwargs.get("status"):
+        status = [kwargs.get("status")]
+
+    qs = (
+        WorkspaceInvitation.objects.filter_for_user(request.user)
+        .filter(status__in=status)
+        .order_by("-updated_at")
+    )
+    return result_page(
+        queryset=qs,
+        page=kwargs.get("page", 1),
+        per_page=kwargs.get("perPage", qs.count()),
```

Comment:
I don't think it's a good idea to return by default the whole list if the users do not give the perPage argument.
Better to have a default value (like "15")

Reply:
Yes, you're right .